### PR TITLE
Added index of contents in README.md

### LINKED ISSUE #320: Add an index to the README.md file
Add an index for the sections of the README.md file.

### PR DESCRIPTION
**Pull request by NotAtitYet**
I have added table of contents in the readme file. Please review and merge my pull request

Resolves #320

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+### Table Of Contents
```
Comment (prestonvasquez) [inline README.md:1]: Can we move this above "Installation" and below the "About" section? And remove the "About Gidari" link.

**Review by prestonvasquez — CHANGES_REQUESTED** [review]
Nice! Just one small request

File: README.md (hunk 1 above)
Comment (NotAtitYet) [reply README.md:1]: I have made the required changes. You can review them.

Comment (NotAtitYet) [reply README.md]: I have made the required changes. You can review them accordingly.

**Comment by prestonvasquez** [comment]
@NotAtitYet Thank you!

**Pull request merged**